packages/workbox-core/_private/cacheWrapper.mjs

```diff
@@ -41,52 +41,56 @@ const putWrapper = async ({
   plugins = [],
   matchOptions,
 } = {}) => {
+  if (process.env.NODE_ENV !== 'production') {
+    if (request.method && request.method !== 'GET') {
+      throw new WorkboxError('attempt-to-cache-non-get-request', {
+        url: getFriendlyURL(request.url),
+        method: request.method,
+      });
+    }
+  }
+
+  const effectiveRequest = await _getEffectiveRequest({
+    plugins, request, mode: 'write'});
+
   if (!response) {
     if (process.env.NODE_ENV !== 'production') {
       logger.error(`Cannot cache non-existent response for ` +
-        `'${getFriendlyURL(request.url)}'.`);
+        `'${getFriendlyURL(effectiveRequest.url)}'.`);
     }
 
     throw new WorkboxError('cache-put-with-no-response', {
-      url: getFriendlyURL(request.url),
+      url: getFriendlyURL(effectiveRequest.url),
     });
   }
 
-  let responseToCache =
-      await _isResponseSafeToCache({request, response, event, plugins});
+  let responseToCache = await _isResponseSafeToCache({
+    response, event, plugins, request: effectiveRequest});
```

Comment:
Nit: I think the old way is technically more correct according to Google JS style.

Reply:
The standalone stuff after `await ...` doesn't fit on a single line anymore, so the old way wouldn't work either.

I'll reformat to put each object property on its own line instead.

Reply:
Yeah, like I said it's a nit, but if you want it on two lines it should be indented 4 spaces (to show it's a line continuation). And if you want to indent 2 spaces it should be expanded (to be a block indent).

The style guide is weird that way, and the eslint plugin doesn't do a good job of handling it.